projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details-table/query-details-table.component.html

```diff
@@ -0,0 +1,38 @@
+<ng-container *ngIf="queryItem">
+  <table class="govuk-table query-details-table" [attr.aria-describedby]="'Details of the query' | rpxTranslate">
```

Comment:
@DavidJayakumar - does `'Details of the query' | rpxTranslate` not need to be surrounded by `{{` and `}}`? I might be wrong here, so could be worth checking with @ronaldmansveld

Reply:
This is correct and also used in multiple other places. @ronaldmansveld  to confirm

Reply:
{{ is only needed in plain text scenarios. This is an attribute within [ and ], thus {{ is not needed.